stable_baselines3/common/policies.py

```diff
@@ -229,20 +228,6 @@ def predict(
         #     mask = [False for _ in range(self.n_envs)]
         observation = np.array(observation)
 
-        # Handle the different cases for images
-        # as PyTorch use channel first format
-        if is_image_space(self.observation_space):
```

Comment:
Same check as with base_class (see if channels are first, and throw warnings/exceptions if not).